script.js

```diff
@@ -0,0 +1,174 @@
+          </ul>
+        </div>
+        <div class="buttons-2">
+          <button class="button-2h" type="button" onclick="showPopup(${project.id})">See Project</button>
+        </div>
+      </div>
+      `;
+    newContainer.insertAdjacentHTML('beforeend', cardsHTML);
+  });
+};
+
+function showPopup(projectId) {
+  const bodyElement = document.querySelector('body');
+  const project = projects.find((pop) => pop.id === projectId);
+  const popupHTML = ` <section id="desk-popup">
+  <div id="desktop-popup">
+     <div> <img src="images/Disabled.png" alt="Disabled" class="pop-close" onclick = "closePopup()" /></div>
+      <div class="pop-image-wrapper">
+          <img src="${project.featured_image}" alt="snapshoot" class="project-image" />
+      </div>
+
+      <div class="pop-heading">
+          <h4 class="paragraph-pop">${project.title}</h4>
+          <div class="pop-button-2h">
+              <a href="${project.live}"><button class="button-mo2">
+                  See Live<img src="images/light.png" alt="live-icon" class="pop-icon" />
+              </button></a>
+              <a href="${project.source}"><button class="button-mo2">
+                  See Source
+                  <img src="images/live.png" alt="git-hub icon" class="pop-icon"/>
+              </button></a>
+          </div>
+      </div>
+       <ul class="button-pop">
+              <li class="list">
+                  <button class="button-pop4">Ruby on Rails</button>
+              </li>
+              <li class="list"><button class="button-pop4">css</button></li>
+              <li class="list">
+                  <button class="button-pop4">Javascript</button>
+              </li>
+              <li class="list">
+                  <button class="button-pop5">codekit</button>
+              </li>
+              <li class="list"><button class="button-pop5">Github</button></li>
+              <li class="list">
+                  <button class="button-pop5">Javascript</button>
+              </li>
+              <li class="list">
+                  <button class="button-pop5">Bootstrap</button>
+              </li>
+              <li class="list"><button class="button-pop5">Terminal</button></li>
+              <li class="list">
+                  <button class="button-pop5">codepen</button>
+              </li>
+      </ul>
+      <p class="paragraph-2 pop-paragraph">
+              ${project.description}
+      </p>
+      <div class="pop-button-3h">
+              <a href="${project.live}"><button class="button-mo1">
+                  See Live<img src="images/light.png" alt="live-icon" class="pop-icon" />
+              </button></a>
+              <a href="${project.source}"><button class="button-mo1">
+                  See Source
+                  <img src="images/live.png" alt="git-hub icon" class="pop-icon" />
+              </button></a>
+          </div>
+    
+  </div>
+</section>`;
+  bodyElement.insertAdjacentHTML('afterbegin', popupHTML);
+}
```

Comment:
Kindly try to match the Figma design for the modal in the mobile version. Please make the close button like the Figma design and make the modal scrollable so the full modal appears.
| Your design | Figma design |
| --- | --- | 
| ![image](https://user-images.githubusercontent.com/24703171/156675454-b7ba0d15-0202-43c6-9b9e-6f6bd679647c.png) | ![image](https://user-images.githubusercontent.com/24703171/156675586-46634e9e-596e-4567-91a3-34b80b2a2a1f.png) |